Projektskabelon-master/src/main/java/FunctionLayer/OverlayMaterialCalculator.java

```diff
@@ -84,7 +86,7 @@ public  static Material screwForOverlayOneWall ( Wall wall, String overlayName)
         screwOverlay.setSize(size);
         screwOverlay.setComment("til montering af beklædningsplanke");
         screwOverlay.setAmount(1);
-
+        screwOverlay.setPrice(LogicFacade.getPrice(screwOverlay.getId()));
```

Comment:
If you set the price here, it will be calculated for the needed size, not for the available one. I calculate de available sizes first in function layer